package.json

```diff
@@ -50,15 +50,23 @@
     "url": "https://github.com/jgilbert01/lambda-api-utils.git"
   },
   "devDependencies": {
+    "@aws-sdk/client-dynamodb": "^3.450.0",
```

Comment:
Ultimately up to you, but it's been my preference lately to try to pin these to the actual version that Lambda runs since we've had issues in the past where we thought some particular attribute was available for use, and then we deploy it and it throws validation errors or we find other issues because the lambda runtime isn't on the latest version of the sdk.

Lambda runtime on both Node 18 and 20 is on 3.515.0. Believe "^3.450.0" currently pulls in 3.556.0.